packages/manager/src/components/TagsInput/TagsInput.spec.js

```diff
@@ -2,89 +2,99 @@ const { navigateToStory } = require('../../../e2e/utils/storybook');
 const { constants } = require('../../../e2e/constants');
 
 describe('Tags Input Suite', () => {
-    const component = 'Tags Input';
-    const childStories = ['Tags Input', 'Tags Input with an error'];
-    const tagInput = '[data-qa-enhanced-select]';
-    const tagOptions = '[data-qa-option]';
-    const selectedTags = '[data-qa-multi-option]';
+  const component = 'Tags Input';
+  const childStories = ['Tags Input', 'Tags Input with an error'];
+  const selectBox = '[data-qa-enhanced-select]';
+  const tagInput = '#add-tags input';
+  const tagOptions = '[data-qa-option]';
+  const selectedTags = '[data-qa-multi-option]';
+  const inputError = '#add-tags-helper-text'
 
-    const inputValidation = (tagText) => {
-      const inputError = $(tagInput).$('..').$('..').$('..').$('p');
-      $(tagInput).$('..').$('input').setValue(tagText);
-      browser.waitForVisible(tagOptions,constants.wait.normal);
-      $$(tagOptions)[0].click();
-      try {
-        inputError.waitForVisible(constants.wait.normal);
-        expect(inputError.getText()).toContain('Length must be 3-50 characters');
-      } catch (e) {
+  const inputValidation = (tagText) => {
+    $(tagInput).setValue(tagText);
+    $(tagOptions).waitForDisplayed(constants.wait.normal);
+    $$(tagOptions)[0].click();
+    try {
```

Comment:
hmm, so what's the purpose for this try/catch.. and should we add logging if in the catch? it seems like it could add a bit of unneeded time to the test.